packages/system/src/Item/DetailView/Tabs/MasterLists.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  MasterListRowFragment,
+  useMasterLists,
+} from '@openmsupply-client/system';
+import { BasicSpinner } from '@common/components';
+import {
+  DataTable,
+  TableProvider,
+  useColumns,
+  createTableStore,
+  Box,
+} from '@openmsupply-client/common';
+
+export const MasterListsTab = () => {
+  const { data, isLoading } = useMasterLists();
```

Comment:
yes, this is nonsense. but it was fun to do 😁

Reply:
Lol. Is there any items that belong to more than one Master list?

Reply:
yeah, reasonably common, though you're right - there will be items on only one list. mSupply currently has the the list of master lists though..